### ISSUE TITLE
NumberField "Cannot read properties of null (reading 'addEventListener')"

### ISSUE THREAD

**Issue by efitzkiwi** [adobe/react-spectrum#5992]
### Provide a general summary of the issue here

The NumberField from react-aria-components does not work, at all. I followed the basic reusable wrapper from https://react-spectrum.adobe.com/react-aria/NumberField.html#reusable-wrappers.

### 🤔 Expected Behavior?

The component should not error when mounting

### 😯 Current Behavior

The component causes the client to crash

### 💁 Possible Solution

NextJS shows this block as the reason

```
  1286 |   let s = r.current;
> 1287 |   return s.addEventListener("beforeinput", n, !1), () => {
       |           ^
  1288 |     s.removeEventListener("beforeinput", n, !1);
  1289 |   };
```

### 🖥️ Steps to Reproduce

1. Install component library
2. Paste wrapper code
3. Client crashes
https://codesandbox.io/p/sandbox/async-water-4rfkxd

### Version

React-aria is 3.32.1, react-aria-components is 1.1.1. I don't have react-spectrum

### What browsers are you seeing the problem on?

Chrome, Microsoft Edge

### What operating system are you using?

Windows 10

**Comment by snowystinger** [comment]
You need to include an `Input`, otherwise your NumberField will have nothing for you to enter text and would be non-functional
```
    <NumberField {...props}>
      <Label>{label}</Label>
      <Group>
        <Button slot="decrement">-</Button>
        <Input />
        <Button slot="increment">+</Button>
      </Group>
      {description && <Text slot="description">{description}</Text>}
      <FieldError>{errorMessage}</FieldError>
    </NumberField>
```

**Comment by efitzkiwi** [comment]
Well, that explains it. I was trying to use a custom input component.
